config/env.development

```diff
@@ -187,22 +187,6 @@ DEPENDENCY_DISCOVERY_URL=http://localhost/v1/dependency-discovery
 # NODE_ENV should be one of "development" or "production"
 NODE_ENV=development
 
-# PORT is the port used by the web server
-PORT=3000
-
-# API_URL is the URL of the Telescope Web API server.  Change this to
```

Comment:
The only thing we lose by doing this is the ability to work on the front-end locally, but use the staging server as your back-end.  Probably that's OK, and we should migrate to using GitPod or EC2 cloud instances instead, so you can run everything at once.